cms/admin/pageadmin.py

```diff
@@ -1221,6 +1246,7 @@ def unpublish(self, request, page_id, language):
             page.unpublish(language)
             message = _('The %(language)s page "%(page)s" was successfully unpublished') % {
                 'language': language_name, 'page': page}
+
```

Comment:
please avoid cosmetic changes unrelated to pr

Reply:
Understood. I'll rectify.